TAF_EasyRest/TestFramework/PageComponents/Client/RestaurantMenuComponents/OrderConfirmationPageComponent.cs

```diff
@@ -0,0 +1,46 @@
+﻿using TestFramework.PageComponents;
+namespace TestFramework.Pages
+{
+    public class OrderConfirmationPageComponent : BasePage
+    {
+        public OrderConfirmationPageComponent(IWebDriver driver) : base(driver)
+        {
+            FillOdersList();
+        }
+
+        #region Elements
+        public List<OrderSummaryDetailsPageComponent> _summaryDetails { get; private set; }
+        private IWebElement _cancelButton => driver.FindElement(By.XPath("//span[text()='Cancel']//parent::button"));
+        private IWebElement _submitButton => driver.FindElement(By.XPath("//span[text()='Submit']//parent::button"));
+        #endregion
+
+        #region Methods
+        private int СountOrders()
+        {
+            IReadOnlyCollection<IWebElement> _summaryDetails = driver.FindElements(By.XPath("//td[@class='MuiTableCell-root-3018 MuiTableCell-body-3020 MuiTableCell-paddingCheckbox-3024']//parent::tr"));
```

Comment:
I can't find these elements. Please check.

Reply:
Agree with Anton. You can't use class names with numbers like this td[@class='MuiTableCell-root-**3018**'] because these numbers are auto-generated during each launch. Try to use instead: td[contains(@class, 'MuiTableCell-root')]